man/systemd-system.conf.xml

```diff
@@ -171,6 +171,126 @@
         effect if a hardware watchdog is not available.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>IOCostQOSPath=</varname></term>
+        <listitem><para> Configure the absolute path to device to be configured
+        in io.cost.qos file of the root cgroup.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>IOCostQOSCtrl=</varname></term>
+        <listitem><para>Configure whether the io.cost.qos parameters are
+        controlled by the kernel or by the user. Allowed values are 'auto' and 'user'.
+        <varname>IOCostQOSRPct</varname>, <varname>IOCostQOSRLat</varname>, <varname>IOCostQOSWPct</varname>,
+        <varname>IOCostQOSRLat</varname>, <varname>IOCostQOSWPct</varname>, <varname>IOCostQOSWLat</varname>
+        <varname>IOCostQOSMin</varname>, and <varname>IOCostQOSMax</varname> are all ignored if value is set to 'auto'. </para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>IOCostQOSEnabled=</varname></term>
+        <listitem><para>Configure whether the io.cost.qos parameters are enabled.
+        Values are 'yes' and 'no' and set to 'no' by default. All other IOCostQOS fields
+        are ignored if this is set to 'no'.
+        </para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>IOCostQOSRPct=</varname></term>
+        <listitem><para>Configure io.cost.qos read latency percentile.
+        Value between 0.00 and 100.00 inclusive accurate to two decimal places.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>IOCostQOSRLat=</varname></term>
+        <listitem><para>Configure io.cost.qos read latency threshold in milliseconds.
```

Comment:
all our time values should always use our time parsers and thus default to µs units unless specified otherwise.